### DISCUSSION TITLE
Multiple custom domains across account, unable to use it on non-primary repo.

### DISCUSSION THREAD

**Discussion by Babyyoda777** [community/community#41343] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,
I have two custom domains on my account, one for the primary .github.io page and another for a .github.io/repo
Both of these are verified, but when I add the second domain to my repo page, it says DNS valid for primary. How would I fix this? The domain which gives the error:
![Screenshot 2022-12-10 at 14 16 47](https://user-images.githubusercontent.com/80623330/206859801-fd67247a-022f-4a73-a8c0-d89b3195b499.png)

**Comment by jfullstackdev** [comment, marked as answer]
this answer might be related 

https://github.com/orgs/community/discussions/22433#discussioncomment-3236678

if you find it problematic, you can always create an Org even if you don't have members and use that for the other custom domain
